travvy/src/components/AttractionDetails.vue

```diff
@@ -1,6 +1,21 @@
 <template>
   <div>
-      <h1>Attraction Details for {{}} </h1>
+    <!-- Button to go to go back to the AttractionsList page --> 
+    <router-link to="/AttractionList" tag="button" class="back"><span>Back</span></router-link>
+
+    <h1>CN Tower </h1>
+    <img src="../images/toronto.jpg" alt="Picture of Toronto"/>
+    <br>
+    <h1>Description</h1>
+    <p>The CN Tower is a 553.3 m-high concrete communications and observation tower located in Downtown Toronto, Ontario, Canada.</p>
+    <h3>Address:</h3>
+    <h3>Email:</h3>
+    <h3>Phone Number:</h3>
+    <h3>Ticket Pricing:</h3>
+    <h3>Maximum Group Size:</h3>
+    <h3>Are masks required:</h3>
```

Comment:
We might want to add other attributes here as well such as start date, close date for the season in future iterations

Reply:
Yeah, that's a good point, I'll definitely account for those once we add them in just to add more details about each attraction!

Reply:
Sounds good!